src/main/routes/browse.ts

```diff
@@ -1,7 +1,38 @@
+import { HTTPError } from 'HttpError';
 import { Application } from 'express';
+import axios from 'axios';
+import config from 'config';
+
+import { PreClient } from '../services/pre-api/pre-client';
+import { SearchCaptureSessionsRequest } from '../services/pre-api/types';
+
 
 export default function (app: Application): void {
-  app.get('/browse', (req, res) => {
-    res.render('browse');
+  app.get('/browse', async (req, res, next) => {
+    try {
+      const client = new PreClient(
+        axios.create({
+          baseURL: config.get('pre.apiUrl'),
```

Comment:
You can move this `baseUrl` and `headers` config to /src/main/app.ts like so:

```typescript
axios.defaults.baseURL = config.get('pre.apiUrl');
axios.defaults.headers.common['Ocp-Apim-Subscription-Key'] = config.get('pre.apiKey');
axios.defaults.headers.common['X-User-Id'] = '1234';
axios.defaults.put['Content-Type'] = 'application/json';
axios.defaults.post['Content-Type'] = 'application/json';
```

Once this is setup you no longer need to pass in an explicit axios instance actually and can call `axios.get...` etc within your `PreClient` without having to initialise it. When testing you can mock out the axios module quite easily which we can go through later